app_secret.go

```diff
@@ -15,10 +15,12 @@ const (
 )
 
 func newAppId(appSecret []byte) []byte {
-	publicKey := appSecret[AppSecretSize-AppPublicKeySize : AppSecretSize]
-	author := make([]byte, 32)
-	payload := append([]byte{appCreationNature}, author...)
-	payload = append(payload, publicKey...)
+	pubKeyStart, pubKeyEnd := AppSecretSize-AppPublicKeySize, AppSecretSize
```

Comment:
Why not one initialization per line, in a `const ()`

Reply:
Can do!